pulp_python/app/models.py

```diff
@@ -42,20 +61,20 @@ class PythonPackageContent(Content):
     name = models.TextField(blank=False)
     version = models.TextField(blank=False)
     metadata_version = models.TextField(blank=False)
-    summary = models.TextField()
-    description = models.TextField()
-    keywords = ArrayField(models.TextField())
-    home_page = models.TextField()
-    download_url = models.TextField()
-    author = models.TextField()
-    author_email = models.TextField()
-    maintainer = models.TextField()
-    maintainer_email = models.TextField()
-    license = models.TextField()
-    requires_python = models.TextField()
-    project_url = models.TextField()
-    platform = models.TextField()
-    supported_platform = models.TextField()
+    summary = models.TextField(null=True, max_length=8192)
```

Comment:
Can you explain the reasoning for this length as a comment, even if it is just an arbitrary large number?

Reply:
I ran into an exception `DataError('value too long for type character varying(64)\n',)`
So to get sync working I just added a rather large max_length.